### DISCUSSION TITLE
The name of the solution

### DISCUSSION THREAD

**Discussion by ShervanN** [ProfessionalCSharp/ProfessionalCSharp2021#129] [General]
Hello,

On page 312, CHAPTER 12 Reflection, Metadata, and Source Generators, You wrote:

> This section starts with the core WhatsNewAttributes .NET library. The source code is contained in the file
> WhatsNewAttributes.cs, which is located in the WhatsNewAttributes project of the **WhatsNewAttributes**
> solution in the example code for this chapter

The name of the solution is **ReflectionSamples** but you wrote above **WhatsNewAttributes**.

**Comment by christiannagel** [comment]
Thanks for reporting this. You're right about the correct solution name.

I've added an update: https://github.com/ProfessionalCSharp/ProfessionalCSharp2021/blob/main/Updates.md#chapter-12-reflection-metadata-and-source-generators
